roles/openshift_control_plane/files/scripts/docker/master-restart

```diff
@@ -15,23 +15,11 @@ for type in "${types[@]}"; do
   fi
 done
 
-# TODO: move to cri-ctl
-# TODO: short term hack for cri-o
-
-# Get a child container name to wait for it to stop
-child_container=$(docker ps -l -q --filter "label=io.kubernetes.container.name=${1}")
-
-container=$(docker ps -l -q --filter "label=openshift.io/component=${1}" --filter "label=io.kubernetes.container.name=POD")
-if [[ -z "${container}" ]]; then
+pod=$(crictl pods -l -q --label "openshift.io/component=${1}" --label "io.kubernetes.container.name=POD")
```

Comment:
```bash
# crictl pods -l -q --label "openshift.io/component=api" --label "io.kubernetes.container.name=POD"
W0531 03:07:48.501263   22407 util_unix.go:75] Using "/var/run/crio/crio.sock" as endpoint is deprecated, please consider using full url format "unix:///var/run/crio/crio.sock".
2a2203551cc3689b7179bb92044d1dd8b122a4b153bdb16f8b02629f570c13ab
```
How about redirect the stderr to /dev/null ?
```bash
# crictl pods -l -q --label "openshift.io/component=${1}" --label "io.kubernetes.container.name=POD" 2>/dev/null

Reply:
I think we should keep stderr so that more debug info would be available

Reply:
@liangxia that's a known warning which is going away as we update crictl.yaml but I'm also fine redirecting stderr.
Usually, if there's an error it's printed to both stderr/stdout